### ISSUE TITLE
Crash when FloatingActionButton.setOnSpeedDialMenuCloseListener() get's null as parameter

### ISSUE THREAD

**Issue by tihomirleka** [markormesher/android-fab#26]
Hi,

I just updated from Java to Kotlin (latest) version of the library and my app crashes when I try to pass `null` to `FloatingActionButton.setOnSpeedDialMenuCloseListener()` method.

I get the following error

```
Fatal Exception: java.lang.IllegalArgumentException: Parameter specified as non-null is null: method kotlin.jvm.internal.Intrinsics.checkParameterIsNotNull, parameter listener
       at uk.co.markormesher.android_fab.FloatingActionButton.setOnSpeedDialMenuCloseListener(FloatingActionButton.kt)
```

I don't know Kotlin so I'm not sure what's the problem.

**Comment by markormesher** [comment]
Hey @tihomirleka - thanks for flagging this! I can see the problem; I'll release a fix shortly.

**Comment by markormesher** [comment]
Fixed in v2.2.1 - let me know if you have any other issues! 👍

**Comment by tihomirleka** [comment]
Just tested it. No crash. Thanks
